File: week1/pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Get positive interger from user
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // Print piramide with height defined by user
    int line;
    for (line = 0; line < height; line++)
    {
        int block = line + 1;
        int space;

        //Print blank spaces according to line specs
        for (space = height - block; space > 0 ; space--)
        {
            printf(" ");
        }

        //Print block spaces according to line specs
        for (block = line + 1; block > 0 ; block--)
        {
            printf("#");
        }

        //Print black space in middle
        printf("  ");

        // Print rest of the block spaces according to line specs
        for (block = line + 1; block > 0; block--)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: week6/pset6/mario.py
from cs50 import get_int


def main():
    n = get_height()
    for i in range(n):
        # Print blank spaces before
        for b in range(n - i - 1):
            print(" ", end="")
        # Print first block column
        for j in range(i + 1):
            print("#", end="")
        # Print blank space between
        print("  ", end="")
        # Print second block column
        for l in range(i + 1):
            print("#", end="")
        print()
    

# Check if number prompted is positive
def get_height():
    while True:
        n = get_int("Height: ")
        if n > 0 and n < 9:
            break
    return n


if __name__ == "__main__":
    main()